src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -28,6 +28,7 @@
     <Compile Include="Helpers.cs" />
     <Compile Include="System\ActivatorTests.cs" />
     <Compile Include="System\ArrayTests.cs" />
+    <Compile Include="System\ArrayTests.netcoreapp1.1.cs" Condition="'$(TargetGroup)' == 'netcoreapp1.1'" />
```

Comment:
I would prefer these all being grouped in one ItemGroup with that ItemGroup having the condition.